Annoucement for Open Science + Research Software Summer School at UIUC in July/August

munkm opened the pull request:
I'm organizing a summer school in collaboration with URSSI on open science + research software. Here's the announcement for the school. 

A few things that should be resolved:
- I said the school was co-hosted by URSSI. Is that the correct language to use here?
- I didn't use a redirect link for the registration form. Is that preferred here? I can add it if so.  
- Much of the research software themes are the same as the existing URSSI schools. I've added language on open science. I'd like to make sure it's clear to a reader that the school covers both open science and research software.

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+---
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills would like to build on and expand them?__
```
```suggestion
__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills and would like to build on and expand them?__
```

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills would like to build on and expand them?__
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign. 
```
```suggestion
In July 2024, we will be hosting a five-day workshop on open science and research software engineering at the University of Illinois Urbana-Champaign. 
```

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five day workshop on open science and research software engineering at the University of Illinois at Urbana-Champaign. 
+
+This workshop complements previous summer and winter schools hosted by URSSI on research software engineering. 
```
maybe add links?

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+
+This is aimed at early-career researchers, particularly graduate students and postdocs, who are familiar with basic skills such as interacting with the Unix shell, version control using Git, and Python programming, and would like to learn more about best practices for developing research software and leveraging their research software to practice and enhance their own open science. 
+
+## Target Audience
```
perhaps say something about being open to all disciplines (if this is right), including humanities and areas that aren't considered "science"?

danielskatz replied:
Or maybe this is just for "science"?

munkm replied:
Thank you so much for pointing this out! I added language saying that all domains are welcome and that we are targeting people who use or develop software in their research.

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+
+## Target Audience
+
+Ideal candidates for this workshop are science practitioners that user or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
```
```suggestion
Ideal candidates for this workshop are science practitioners who use or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
```

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+Ideal candidates for this workshop are science practitioners that user or develop software in their research and want to share their software in their community of practice, or are contributing to other research projects. 
+These practitioners want to make their research open, accessible, and reproducible by implementing open science best practices in addition to building and contributing to research software. 
+
+To get the most benefit from this workshop, we expect students to be familiar with the Unix shell, Python, and git, at the level taught at a Software Carpentry Workshop. 
```
maybe link to SC?

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+*   Software, Data, and Documentation Licensing
+*   Reproducibility
+
+The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant will bring a project to work on throughout the course for applying these concepts.
```
```suggestion
The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant should bring a project to work on throughout the course for applying these concepts.
```

danielskatz replied:
or "must"?

munkm replied:
What do you think about "is strongly encouraged to"?

danielskatz replied:
sure. Maybe also say this earlier, such as by line 16, to emphasize it

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+## Logistics
+
+*   **Dates**: July 29 - August 02, 2024
+*   **Location**: Urbana-Champaign, IL - [University of Illinois at Urbana-Champaign](https://illinois.edu)
```
```suggestion
*   **Location**: Champaign-Urbana, IL - [University of Illinois Urbana-Champaign](https://illinois.edu)
```

munkm replied:
Do you think switching the order of the cities is confusing? Especially if it's the other order in the school name?

danielskatz replied:
yes, but it's not our fault :)

munkm replied:
Ok! I changed it to Urbana, as all of the lessons will be held on the Urbana side of Wright St. 🙂

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+
+*   **Dates**: July 29 - August 02, 2024
+*   **Location**: Urbana-Champaign, IL - [University of Illinois at Urbana-Champaign](https://illinois.edu)
+*   **Cost**: Free (supported by a grant from the NASA Transform to Open Science Training call)
```
again, maybe add a link?

danielskatz commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,55 @@
+## Important Dates
+
+*   **Registration deadline**: 06 June 2024
+*   **Notification**: June 10
```
```suggestion
*   **Notification**: 10 June
```

danielskatz replied:
or consistently the other way in this line and previous one

munkm commented:
Thank you so much for your thorough review @danielskatz ! This post is significantly improved with your suggestions. 

I've done the following:
- Made sure dates are consistently stated 
- Added links to sw carpentry, the previous URSSI workshops, and to the TOPS-T announcement blog post 
- Added language about the target audience
- Added enhancing language about the project + collaboration aspect of the workshops.

munkm commented:
Here's a preview for the blog post: https://deploy-preview-52--urssi.netlify.app/blog/2024/05/20/applications-now-open-for-a-summer-school-in-open-science--research-software-engineering/

danielskatz reviewed (approved):
lgtm

kyleniemeyer commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,59 @@
+
+This workshop complements previous [summer](https://urssi.us/blog/2024/02/29/applications-now-open-for-the-2024-urssi-summer-school-in-research-software-engineering/) and [winter](https://urssi.us/blog/2023/10/31/applications-now-open-for-the-2024-urssi-winter-school-in-research-software-engineering/) schools hosted by URSSI on research software engineering. 
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering. 
+To that end, we strongly encourage attendees to consider a particular software project to apply learning principles to during the school. 
```
The existing software lessons sort of rely on people coming with something to work on - so you may want to make bringing a project (or multiple things!) mandatory and not just strongly suggested

kyleniemeyer commented on content/blog/2024-05-20-summer-software-open-science-school.md:
```diff
@@ -0,0 +1,59 @@
+*   Software, Data, and Documentation Licensing
+*   Reproducibility
+
+The school will consist of lectures on these topics along with open hacking time to allow participants to practice the concepts covered in the lectures. To facilitate the hands-on experience, each participant is strongly encouraged to bring a project to work on throughout the course for applying these concepts.
```
same comment as above - I recommend making bringing a project or things to work on mandatory (and part of the application)
